.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+#### What does this PR do?
```

Comment:
@gildniy you should not fill in the pull request here. This should be empty and will autofill your pull request text area when you create a new pull request.